From 91632ebf9ef57f2c1aeb6349e80d2af22f862694 Mon Sep 17 00:00:00 2001
From: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Date: Tue, 12 Jun 2018 10:01:45 +0300
Subject: [PATCH 3956/5242] MLK-18562 sound: asoc: remove micfil unsupported
 rates and formats

commit  628953dd1d2976e6022376bb9d267fe1d96c38f4 from
https://source.codeaurora.org/external/imx/linux-imx.git

This block implements the required digital interface to provide a
16-bit audio signal from a PDM microphone bitstream in a
configurable output sampling rate 11025kHz - 48000kHz.

Signed-off-by: Cosmin-Gabriel Samoila <cosmin.samoila@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_micfil.c |    7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/sound/soc/fsl/fsl_micfil.c b/sound/soc/fsl/fsl_micfil.c
index 5eb3a91..6e96635 100644
--- a/sound/soc/fsl/fsl_micfil.c
+++ b/sound/soc/fsl/fsl_micfil.c
@@ -25,11 +25,8 @@
 #include "fsl_micfil.h"
 #include "imx-pcm.h"
 
-#define FSL_MICFIL_RATES		SNDRV_PCM_RATE_8000_192000
-#define FSL_MICFIL_FORMATS		(SNDRV_PCM_FMTBIT_S8 | \
-					SNDRV_PCM_FMTBIT_S16_LE | \
-					SNDRV_PCM_FMTBIT_S20_3LE | \
-					SNDRV_PCM_FMTBIT_S24_LE)
+#define FSL_MICFIL_RATES		SNDRV_PCM_RATE_8000_48000
+#define FSL_MICFIL_FORMATS		(SNDRV_PCM_FMTBIT_S16_LE)
 
 struct fsl_micfil {
 	struct platform_device *pdev;
-- 
1.7.9.5

